src/modules/new-request-form/translations/en-us.yml

```
@@ -0,0 +1,134 @@
+      value: "View article"
+  - translation:
+      key: "new-request-form.answer-bot-modal.footer-title"
+      title: "Title of the footer of the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Does this article answer your question?"
+  - translation:
+      key: "new-request-form.answer-bot-modal.footer-content"
+      title: "Title of the footer of the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "If it does, we can close your recent request #{{requestId}}"
+  - translation:
+      key: "new-request-form.answer-bot-modal.mark-irrelevant"
+      title: "Label of the button used to mark a suggested article as irrelevant in the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "No, I need help"
+  - translation:
+      key: "new-request-form.answer-bot-modal.solve-request"
+      title: "Label of the button used to close the request in the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Yes, close my request"
+  - translation:
+      key: "new-request-form.answer-bot-modal.request-submitted"
+      title: "Notification used in the Answer Bot Modal to inform the user that their request was submitted"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Your request was successfully submitted."
+  - translation:
+      key: "new-request-form.answer-bot-modal.request-closed"
+      title: "Notification shown after a request has been closed from the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/1ypC8cFgu198OkiUkr9I0cqtzIjqZl9PQ/view?usp=drive_link"
+      value: "Nice! Your request has been closed."
+  - translation:
+      key: "new-request-form.answer-bot-modal.solve-error"
+      title: "Notification shown when there is an error closing a request from the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/1AJxNZ6HNoFbANJiRSfzOxUTGfJ2XYqyH/view?usp=sharing"
+      value: "Oops! There was an error closing your request."
+  - translation:
+      key: "new-request-form.dropdown.empty-option"
+      title: "[A11Y] Hidden label for screen readers, describing an empty option in a dropdown"
+      screenshot: "https://drive.google.com/file/d/1enaOJceIEIVn_uYqvaF4AHlE1_RIpjqe/view?usp=drive_link"
+      value: "Choose an option"
+  - translation:
+      key: "new-request-form.suggested-articles"
+      title: "Title for the suggested articles section"
+      screenshot: "https://drive.google.com/file/d/1enaOJceIEIVn_uYqvaF4AHlE1_RIpjqe/view?usp=drive_link"
+      value: "Suggested Articles"
```

```suggestion
      value: "Suggested articles"
```
CD wants us to use the sentence case even for headings: https://garden.zendesk.com/content/grammar#capitalization